app/src/main/java/com/example/calculator_final/DesignActivity.kt

```diff
@@ -0,0 +1,115 @@
+package com.example.calculator_final
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentTransaction
+import kotlinx.android.synthetic.main.activity_constraints.*
+import net.objecthunter.exp4j.ExpressionBuilder
+
+class DesignActivity : AppCompatActivity(), Communicator {
+
+    companion object {
+        const val TAG: String = "TAG"
+    }
+
+    var isFragmentOneDisplay: Boolean = false
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_constraints)
```

Comment:
```suggestion
        setContentView(R.layout.activity_design)
```
Xem lại coding convention nhé em
https://github.com/framgia/coding-standards/blob/master/eng/android/standard.md

Reply:
Vâng ạ, lắm lỗi quá  huhu :(